### PR TITLE
New upgrade tokens need security analysis

### PR DESCRIPTION
**Pull request by DavidSchinazi**
Written in response to [Roman's COMMENT](https://mailarchive.ietf.org/arch/msg/masque/16NJtUKTP6FMK-AuGjhbQfujS9k/)

### CODE REVIEW

**Comment by DavidSchinazi** [comment]
From [Roman](https://mailarchive.ietf.org/arch/msg/masque/G92KvVy8uR2deTCqtle5x8Bkb8s/) in reference to the text in this PR:
> No problem.  Your alternative text captures the spirit of what I was intending.  I was focused on reinforcing the idea that an end-to-end re-analysis will be needed when the Capsule protocol/HTTP Datagrams are integrated into another protocol.

File: draft-ietf-masque-h3-datagram.md [hunk 1]
```diff
@@ -512,6 +512,10 @@ Since use of the Capsule Protocol is restricted to new HTTP Upgrade Tokens, it
 is not accessible from Web Platform APIs (such as those commonly accessed via
 JavaScript in web browsers).
 
+Definitions of new HTTP Upgrade Tokens that use the Capsule Protocol need to
+perform an appropriate security analysis that considers the impact of HTTP
+Datagrams and Capsules in the context of their protocol.
+
```
Comment (tfpauly) [inline draft-ietf-masque-h3-datagram.md:518]:
```suggestion
```

**Review by tfpauly — APPROVED** [review]

File: draft-ietf-masque-h3-datagram.md (hunk 1 above)
Comment (tfpauly) [reply draft-ietf-masque-h3-datagram.md:518]: Cool!

File: draft-ietf-masque-h3-datagram.md [hunk 2]
```diff
@@ -512,6 +512,10 @@ Since use of the Capsule Protocol is restricted to new HTTP Upgrade Tokens, it
 is not accessible from Web Platform APIs (such as those commonly accessed via
 JavaScript in web browsers).
 
+Definitions of new HTTP Upgrade Tokens that use the Capsule Protocol need to
+perform an appropriate security analysis that considers the impact of HTTP
```
[the remark this answers is not in the document]
Comment (DavidSchinazi) [reply draft-ietf-masque-h3-datagram.md]: Done

**Pull request merged**